webpack.config.babel.js

```diff
@@ -176,6 +176,7 @@ const DEVELOPMENT_CONFIG = {
   PUBLIC_PATH: '/',
   MOBILE_URL: '/',
   RESTRICTED_XDEVICE_FEATURE_ENABLED: false,
+  SDK_CONFIGURATION: {},
```

Comment:
Please backfill this with default configuration of disabled multi-frame capture 🙏

Reply:
It's planned in the MFC PR, but I can do it now.

Reply:
Ah okay then. You can keep this for a minimal diffs 🙏